py-polars/polars/internals/dataframe/frame.py

```diff
@@ -274,13 +274,13 @@ def __init__(
         elif isinstance(data, dict):
             self._df = dict_to_pydf(data, columns=columns)
 
-        elif isinstance(data, Sequence) and not isinstance(data, str):
-            self._df = sequence_to_pydf(
-                data, columns=columns, orient=orient, infer_schema_length=50
-            )
         elif isinstance(data, pli.Series):
             self._df = series_to_pydf(data, columns=columns)
 
+        elif isinstance(data, (list, tuple, Sequence)):
```

Comment:
should it have the `not isinstance(data, str)` check or is it enough that it is handled in sequence_to_pydf?

Reply:
Turns out it's actually [fine](https://github.com/pola-rs/polars/pull/6064#discussion_r1062412379) to drop it :)